### DISCUSSION TITLE
A fabric terminal for Windows platform

### DISCUSSION THREAD

**Discussion by spampinato55** [danielmiessler/Fabric#1067] [Show and tell]
At https://github.com/spampinato55/FabricTerminalWindows  a Windows project in C#/WPF. Its purpose is to make it easier to use 'fabric' for people who aren't comfortable with a terminal .
At https://sites.google.com/view/caf-etool-2020/fabric-terminal-for-windows a short explanation.

**Comment by Stefyo1977** [comment]
there is no .exe file

**Comment by spampinato55** [comment]
Links refer to the project in C#, the application is at [https://bit.ly/4gYEGqo](url). It is a .zip folder, extract it and you find the FabricTermila2.exe . Please if you test it make know the results.
